### PR TITLE
Components

### PR DESCRIPTION
**Pull request by YuvrajM108**
In this pull request I did the following:
- Create a calculator component and a button component.
- Add HTML elements to the render methods of respective components.
- Add basic styling for elements.
- Correct all compilation errors.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,7 +11,4 @@ ReactDOM.render(
   document.getElementById('root'),
 );
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
 reportWebVitals();
```
Comment (Georjane) [inline src/index.js]: - [x] You can go ahead and delete all the boilerplate code that got created by the CRA (texts, images, and styles). Like these here. 
- [x] Don't forget the files too and images. You won't be needing them for your project

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,11 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>Math Magicians</h1>
+      <Calculator />
     </div>
```
Comment (Georjane) [inline src/App.js]: - [x] Your `App.js` component should be class-based, just like you did with the calculator and button components

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Button from './Button';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="calculator">
```
Comment (Georjane) [inline src/components/Calculator.js:8]: - [x] The `Calculator.js` component should match the following given design:
[https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/calculator.png](image)

File: src/App.css [hunk 4]
```diff
@@ -1,39 +1,49 @@
...
+  justify-content: center;
+  align-items: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.calculator {
+  width: 80%;
 }
```
Comment (Georjane) [inline src/App.css]: - [x] In order to match the required calculator design, I suggest you add a `max-width` property to the calculator class

| Required Design            | Your Design   |
| ------------------- | -------------- |
| image  | image    |

**Review by Georjane — CHANGES_REQUESTED** [review]
### Hello, @YuvrajM108, ✋🏽 

Good job so far! Your Readme file and PR are professional. Your linters are all passing, I applaud that. 👏🏽 
However, we have a few more changes to make in order to meet all requirements for this project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding! 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝 

_Code Reviewer
**[Jane](https://witahgeorjane.tech/)**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (DeliceLydia) [reply src/index.js]: Addressed this but you can just keep `index.html` in the `public` folder I think that what you need for now

File: src/App.js (hunk 2 above)
Comment (DeliceLydia) [reply src/App.js]: Addressed this

File: src/components/Calculator.js (hunk 3 above)
Comment (DeliceLydia) [reply src/components/Calculator.js:8]: Design looks okey

File: src/App.css (hunk 4 above)
Comment (DeliceLydia) [reply src/App.css]: Done this!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉🎉 

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**